.github/workflows/conformance-gke.yaml

```diff
@@ -152,6 +157,7 @@ jobs:
             --helm-set=hubble.relay.image.repository=quay.io/${{ github.repository_owner }}/hubble-relay-ci \
             --helm-set=hubble.relay.image.tag=${SHA} \
             --helm-set=agentNotReadyTaintKey=ignore-taint.cluster-autoscaler.kubernetes.io/cilium-agent-not-ready \
+            --helm-set=debug.enabled=true \
```

Comment:
i guess this is unrelated?

Reply:
correct, that's why it's on a different commit

Reply:
is it something specific to GKE, or do we want debug enabled across all the conformance tests?

Reply:
Probably all of them since it was helpful for me to understand what was going on with the current GKE failure.